pypesto/petab/importer.py

```diff
@@ -78,6 +81,74 @@ def from_yaml(yaml_config: Union[dict, str],
             output_folder=output_folder,
             model_name=model_name)
 
+    def check_gradients(
+        self,
+        *args,
+        rtol: float = 1e-2,
+        atol: float = 1e-3,
+        mode: Literal = None,
+        objective: Objective = None,
+        objAbsoluteTolerance: float = 1e-10,
+        objRelativeTolerance: float = 1e-12,
```

Comment:
```suggestion
        obj_absolute_tolerance: float = 1e-10,
        obj_relative_tolerance: float = 1e-12,
```

We use snake_case for everything but class names (PEP8). Please also adapt below.